### ISSUE TITLE
No animation when cursor is placed before some added characters

### ISSUE THREAD

**Issue by nikunjdaga** [alphamu/AnimatedEditText#6]
AnimatedEditText behaves as a normal EditText if you place cursor on the left of some added characters and then type again

**Comment by alphamu** [comment]
Yes, this is by design. Unfortunately, the code for handling this was getting extremely complex so I decided to cut it. That said, the widget is dude for a major upgrade so I'll consider adding it back in.
